network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -27,6 +27,11 @@
     {% include "./fragments/homepage_heroguts.html" with root=root page=page homepage=True %}
 {% endblock %}
 
+{% block donate_banner %}
+    {% if page.donate_banner %}
+        {% include "fragments/donate_banner.html" with banner=page.donate_banner.localized %}
+    {% endif %}
+{% endblock %}
```

Comment:
Updating the homepage template to render the donate banner it has set for it's `donate_banner` field.